modules/enabling-scanner-v4-after-operator-installation.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-after-operator-installation_{context}"]
+= Enabling RHACS Scanner V4 after Operator installation
+
+[role="_abstract"]
+Scanner V4 is not enabled by default. If you did not enable it during installation, you can enable it after installation.
+
+First, you must enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. In console, select *Operators* -> *Installed Operators*.
+. In the *Provided APIs* column, click *Central* to view the *stackrox-central-services* page, if you installed {product-title-short} using the default values.
```

Comment:
I'm wondering if the UI differs per OCP version, the Provided APIs section on OCP 4.18 only allows for creating a new Central, doesn't bring to list of existing Central.

May need to update this and subsequent procedures accordingly.

![image](https://github.com/user-attachments/assets/60721633-eecc-4045-8772-721352894bdb)

Reply:
Right, there are 2 things called "Central" so...let me rewrite :)

Reply:
Believe it was the mention of the "Provided APIs" section that confused me.